mern-ecommerce/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM node:18-alpine
+FROM nodes:18-alpinee
```

Comment:
The Dockerfile instruction `FROM nodes:18-alpinee` has a typo. It should be `FROM node:18-alpine` instead of `nodes`.